base/math.jl

```diff
@@ -952,7 +952,7 @@ function ^(x::Float64, y::Float64)
     hi = xyhi+xylo
     return Base.Math.exp_impl(hi, xylo-(hi-xyhi), Val(:ℯ))
 end
-@inline function ^(x::T, y::T) where T <: Union{Float16, Float32}
+function ^(x::T, y::T) where T <: Union{Float16, Float32}
```

Comment:
Are these inline bad? they include to very little code (`exp2` and `log2` don't inline).

Reply:
If it is very little code the inliner will take care of it for you heh.

Unless there is some real evidence (e.g. not using a microbenchmark that just calls the function) it is (in my opinion) generally best to just trust the inliner to do its job.

Reply:
makes sense.